src/main/java/nextstep/subway/domain/station/StationRegisterStatus.java

```diff
@@ -27,4 +29,64 @@ public void validate(StationPosition stationPosition, Distance distance, Station
                     stationStatus.splitLineStation(stationPosition, interStation, distance);
                 });
     }
+
+    public LineStation createLineStation() {
+        if (lineStationCreateValidate()) {
+            Station upStation = getStationByPosition(StationPosition.UPSTATION);
+            Station downStation = getStationByPosition(StationPosition.DOWNSTATION);
+            Distance distance = distanceSum();
+            Line line = commonLine();
+            return new LineStation(downStation, upStation, distance, line);
+        }
+        return null;
+    }
+
+    private boolean lineStationCreateValidate() {
+        return validateUpDownStationExist() && allPositionNotNone() && allLineMatch();
+    }
+
+    private boolean validateUpDownStationExist() {
+        long upStationCount = stationStatusList.stream()
+                .filter(stationStatus -> stationStatus.matchStationPosition(StationPosition.UPSTATION))
+                .count();
+        long downStationCount = stationStatusList.stream()
+                .filter(stationStatus -> stationStatus.matchStationPosition(StationPosition.DOWNSTATION))
+                .count();
```

Comment:
```suggestion
        long upStationCount = getCount(StationPosition.UPSTATION);
        long downStationCount = getCount(StationPosition.DOWNSTATION);
```

반복되는 로직을 메소드로 분리해보면 좋을것 같네요!

Reply:
중복제거! 😱  코드가 길면 자연스레 의식하고 중복제거해야겠다 생각하는데 아직 디테일한 부분까지는 잡아내지 못하네요 ☔ 
리팩토링 단계에서 잡아낼 수 있도록 더 신경써서 보는 습관 길러봐야겠습니다! 감사합니다 😀